packages/web/src/js/Toast.ts

```diff
@@ -40,7 +40,12 @@ const SELECTOR_CLOSE_BUTTON_ELEMENT = `[${ATTRIBUTE_DATA_POPULATE_FIELD}="close-
 const SELECTOR_DISMISS_TRIGGER_ELEMENT = `[${ATTRIBUTE_DATA_DISMISS}="${NAME}"]`;
 const SELECTOR_MESSAGE_ELEMENT = `[${ATTRIBUTE_DATA_POPULATE_FIELD}="message"]`;
 
-export const SLOWEST_TRANSITION_PROPERTY_NAME = 'max-height'; // Keep in sync with transitions in `scss/Toast/_theme.scss`.
+// Keep in sync with transitions in `scss/Toast/_theme.scss`.
+export const SLOWEST_TRANSITION_PROPERTY_NAME = {
```

Comment:
What about reversing the naming to `PROPERTY_NAME_SLOWEST_TRANSITION`. It is the same naming as it is above with the `SELECTOR_`.